### PR TITLE
676 add extracted HH model

### PR DESCRIPTION
**Pull request by mwfrost**
Ticket #676 adds a new data group type for HHE Reports. This branch includes numerous changes to the seed data to catch up with the current schema.

### CODE REVIEW

File: dashboard/tests/functional/test_datadocument_detail.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def test_absent_extracted_text(self):
 
     def test_script_links(self):
         doc = DataDocument.objects.first()
-        response = self.client.get(f'/datadocument/{doc.pk}/')
+        #response = self.client.get(f'/datadocument/{doc.pk}/')
```
Comment (debboutr) [inline dashboard/tests/functional/test_datadocument_detail.py]: there is no need for the line above if we are going to get an explicit URL

File: dashboard/models/extracted_hhrec.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    sampling_method = models.TextField("Sampling Method", 
+                                        null=True, blank=True)
+    analytical_method = models.TextField("Analytical Method", 
+                                        null=True, blank=True)
+    num_measure = models.CharField("Numeric Measure", max_length=50,
+                                        null=True, blank=True)
+    num_nondetect = models.CharField("Numeric Nondetect", max_length=50,
+                                        null=True, blank=True)                                    
+
+    rawchem_ptr = models.OneToOneField(blank=False, null=False,
```
Comment (debboutr) [inline dashboard/models/extracted_hhrec.py]: this field does not necessarily need to be expressed in the model def this way, the only thing this changes in the field is the `auto_created` attribute. If we want to be consistent we should either blow all of these defs out of child models OR add the `extractedcpcat_ptr` to ExtractedListPresence

File: dashboard/models/extracted_hhrec.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+from .raw_chem import RawChem
+from .extracted_text import ExtractedText
+
+class ExtractedHHRec(TimeStampedModel, RawChem):
```
Comment (debboutr) [inline dashboard/models/extracted_hhrec.py]: I'm guessing this gives us `created_at` and `updated_at` in the model? If we like how this works are we going to inherit this into other models or continue to use both?

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: Yes, it would replace our CommonInfo base class.

File: dashboard/models/extracted_hhrec.py (hunk 2 above)
Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: You are right - if you use the default name for the parent key on a subclass, you don't need to declare anything. I just deleted that explicit declaration from the model, made a migration to remove it, and then ran makemigrations. When it finished up, the column was still in the table where it needed to be.

File: dashboard/models/extracted_hhrec.py (hunk 3 above)
Comment (zachmunzenrider) [reply dashboard/models/extracted_hhrec.py]: Might be good to have a ticket to replace CommonInfo across the board. Or would that be too big a PITA?

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: If we decide to do that, it deserves its own ticket. I don't think it would be too much work.

Comment (zachmunzenrider) [reply dashboard/models/extracted_hhrec.py]: Yes, definite;ly its own ticket! I'm just wondering if the refactoring is worth it

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: It occurs to me now that CommonInfo records the user who updates the record - TimeStampedModel does not. That could be a dealbreaker. I should probably roll this back to CommonInfo and redo the seed data and migrations accordingly.

Comment (zachmunzenrider) [reply dashboard/models/extracted_hhrec.py]: I'd forgotten that! I do think it's worthwhile to capture the user. Maybe we should make CommonInfo a TimeStampedModel ;-)

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: Looking again at CommonInfo, it does _not_ include the updating user. We are only recording that ad hoc in different forms. What we really need is a TimeAndUserStampedModel abstract base class.

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: But it's clear that I have just complicated things by using an alternative to CommonInfo and I'll revert to that class.

Comment (zachmunzenrider) [reply dashboard/models/extracted_hhrec.py]: Cool. It looks like Rick and I are both reviewing this ticket? I'll hold off until you get this checked in and then look again

Comment (mwfrost) [reply dashboard/models/extracted_hhrec.py]: Thanks. I am going to contain all the changes in the 0090 migration

File: dashboard/tests/functional/test_datadocument_detail.py (hunk 1 above)
Comment (mwfrost) [reply dashboard/tests/functional/test_datadocument_detail.py]: I'd like to clean this up later when I write tests against the data document detail view.

**Pull request merged**